### PR TITLE
Refracted/Reflected reconstruction maps

### PR DESCRIPTION
**Pull request by AlanSalcedo**
Added interferometric maps for refracted/reflected solutions for distant solutions.

### CODE REVIEW

File: araproc/analysis/standard_reco.py [hunk 1]
```diff
@@ -236,34 +243,81 @@ def do_standard_reco(self, waveform_bundle):
...
+            "radius": self.distant_events_r_library[1]
+        }
+
+        # Store the refracted/reflected rays results
+        reco_results["distant_v_ref"] = {
+            "corr": corr_distant_v_ref, 
+            "theta": theta_distant_v_ref,
+            "phi": phi_distant_v_ref,
+            "map": distant_map_v_ref,
+            "radius": self.distant_events_r_library[1]
```
Comment (clark2668) [inline araproc/analysis/standard_reco.py:283]: I would prefer if you did `self.rtc_wrapper.correlators["distant"].GetRadius()`

Comment (clark2668) [reply araproc/analysis/standard_reco.py:283]: That way the only thing that's important is that you have your handle on the right correlator. It's more modular (I think...)

**Review by clark2668 — CHANGES_REQUESTED** [review]

File: araproc/analysis/standard_reco.py (hunk 1 above)
Comment (AlanSalcedo) [reply araproc/analysis/standard_reco.py:283]: Ah, I didn't know I could get the radius from there. Thanks!

Comment (clark2668) [reply araproc/analysis/standard_reco.py:283]: Yup! :)

Comment (AlanSalcedo) [reply araproc/analysis/standard_reco.py]: Done. Should I merge or wait for @marcomuzio?

Comment (marcomuzio) [reply araproc/analysis/standard_reco.py]: Merge away!

**Pull request merged**